demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1013,51 +1021,36 @@ def is_context_change_in_readme(self) -> bool:
             True if there has been a corresponding change to README file when context is changed in integration
         """
         valid = True
-        # the pattern to get the context part out of command section:
-        context_section_pattern = CONTEXT_OUTPUT_README_TABLE_HEADER.replace('|', '\\|').replace('*', r'\*') + ".(.*?)#{3,5}"
-        # the pattern to get the value in the first column under the outputs table:
-        context_path_pattern = r"\| ([^\|]*) \| [^\|]* \| [^\|]* \|"
 
         dir_path = os.path.dirname(self.file_path)
         if not os.path.exists(os.path.join(dir_path, 'README.md')):
             return True
 
-        # get README file's content
         readme_path = os.path.join(dir_path, 'README.md')
+        if f'{readme_path} - [RM102]' in FOUND_FILES_AND_IGNORED_ERRORS \
+                or f'{readme_path} - [RM102]' in FOUND_FILES_AND_ERRORS \
+                or f'{self.file_path} - [IN136]' in FOUND_FILES_AND_IGNORED_ERRORS \
+                or f'{self.file_path} - [IN136]' in FOUND_FILES_AND_ERRORS:
+            return False
```

Comment:
I thought that if we are ignoring these errors in `handle_error`. so why the skip here?

Reply:
After consulting with @roysagi, we have decided this is the best option to avoid duplicated errors.
I will change the error's codes to dynamic codes, so there will be no dependency on the error's code.